### PR TITLE
feat(Dev/loader UI):Added loader in edit,log work pages,create task, sprint and organization

### PR DESCRIPTION
**Pull request by Pendyala-thanvi**
### Functionality:
Added a loader.
### Solution:
Added a loader in edit, log work page, create task, sprint and organization.

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

### CODE REVIEW

**Comment by viv9k** [comment]
Multiple PR is there for same issue #200 

@shivamgoyal7 @k0malSharma Please get this fixed and get the task completed asap.

**Comment by Pendyala-thanvi** [comment]
> Multiple PR is there for same issue #200
> 
> @shivamgoyal7 @k0malSharma Please get this fixed and get the task completed asap.

> @viv9k Both the PR's have different changes and the names are edited according to tasks.

File: src/app/body/create-new-session/create-new-session.component.ts [hunk 1]
```diff
@@ -109,15 +109,16 @@ export class CreateNewSessionComponent implements OnInit {
...
     } catch (error) {
       this.errorHandlerService.getErrorCode(this.componentName, "InternalError");
       this.enableLoader = false;
     }
   }
 
   backToDashboard() {
-    this.location.back()
+    this.router.navigate(['MyDashboard']);
+    //this.location.back()
```
Comment (arjund-06) [inline src/app/body/create-new-session/create-new-session.component.ts:121]: I dont think this change is required in both create sprint and create new task

File: src/app/body/create-new-sprint/create-new-sprint.component.ts [hunk 2]
```diff
@@ -135,7 +135,8 @@ export class CreateNewSprintComponent implements OnInit {
   }
 
   backToDashboard() {
-    this.location.back();
+    this.router.navigate(['MyDashboard']);
+    //this.location.back();
```
Comment (arjund-06) [inline src/app/body/create-new-sprint/create-new-sprint.component.ts:139]: Here too

File: src/app/body/create-new-session/create-new-session.component.ts (hunk 1 above)
Comment (Pendyala-thanvi) [reply src/app/body/create-new-session/create-new-session.component.ts]: I removed those lines from the file.

**Review by arjund-06 — APPROVED** [review]
Ship it

**Review by viv9k — APPROVED** [review]
Ship it

**Pull request merged**